apm-sniffer/apm-sdk-plugin/spring-plugins/mvc-annotation-commons/src/main/java/org/apache/skywalking/apm/plugin/spring/mvc/commons/SpringMVCPluginConfig.java

```diff
@@ -35,16 +37,35 @@
              * request.
              */
             public static boolean COLLECT_HTTP_PARAMS = false;
+
+            /**
+             * This config item controls that whether the SpringMVC plugin should collect the headers of the
+             * request.
+             */
+            public static boolean COLLECT_HTTP_HEADERS = false;
         }
 
         @PluginConfig(root = SpringMVCPluginConfig.class)
         public static class Http {
             /**
              * When either {@link Plugin.SpringMVC#COLLECT_HTTP_PARAMS} is enabled, how many characters to keep and send
              * to the OAP backend, use negative values to keep and send the complete parameters, NB. this config item is
-             * added for the sake of performance
+             * add for the sake of performance
```

Comment:
Why modify this

Reply:
grammer wrong

Reply:
What do you mean `grammer wrong`? `is added` is a correct expression.

Reply:
I think we should use , was added , or is add,  the grammer check is wrong in IDEA

Reply:
soory, my fault will revert this change

Reply:
I don't think so. The comments don't use `past tense`, because it describes the codes currently are for. Also, this is the object to be added, so clearly, `is added` is correct.
I don't know why the IDE says so, but from my understanding in English, it is not an issue. IDE is not the standard for a speaking language.

Reply:
yes you are right